src/dag/dag_block_manager.cpp

```diff
@@ -245,12 +245,14 @@ DagBlock DagBlockManager::popVerifiedBlock(bool level_limit, uint64_t level) {
       cond_for_verified_qu_.wait(lock);
     }
   }
-  if (stopped_) return DagBlock();
+  if (stopped_) return {std::make_shared<DagBlock>(), false};
```

Comment:
maybe nullptr could be returned here and caller should check the value to nullptr ?

Reply:
sure, I will change this